data/human/hails.txt

```diff
@@ -1982,24 +1974,157 @@ phrase "friendly civilian"
 		"Don't get on the bad side of the authorities. "
 		"I always try to be an honest captain. "
 		"Crime doesn't pay in the end. "
-		""
+		"You've got to be careful if you associate yourself with the wrong sorts. "
+		"" 4
 	word
 		"It's a fact"
 		"I've heard"
-		"I heard a rumor"
 		"You should know"
 		"They say"
 		"It's said"
+		"I know from experience"
+		"Once you've been on the starlanes long enough you know"
 	word
 		" that "
 	word
-		"smugglers almost always get caught in the end."
-		"even the most powerful warship won't save you from a knife in the back."
-		"the Quarg will arrest you if they catch a whiff of any of their equipment on your ship."
-		"the Quarg imprison anyone they catch carrying their outfits."
-		"nine out of ten pirates die within their first year."
-		"Republic Intelligence has their eyes everywhere."
-		"the law will always catch up to those who try to bypass it."
+		"smuggling is a sure way to find yourself behind bars"
+		"even the most powerful warship won't save you from a knife in the back"
+		"nine out of ten pirates die within their first year"
+		"Republic Intelligence has their eyes everywhere"
+		"the law will always catch up to those who try to bypass it"
+	word
+		"."
+
+phrase "friendly civilian"
+	word
+		"The Quarg are pretty strict with their technology. They"
+		"I wish the Quarg would share some of their technology with us, but they"
+		"You don't want to be caught with Quarg technology around Enif. The Quarg"
+		"The Quarg don't mess around with their technology. They"
+		"I'd love to get my hands on some Quarg tech, but they"
+		"It's said that the Quarg"
+	word
+		" make it pretty clear that they "
+		" monitor their ships and outfits meticulously. They "
+		" don't want their advanced technology to proliferate, and they "
+		" restrict their outfits to their own ships, and "
+		" " 4
+	word
+		"will arrest you if they catch a whiff of any of their equipment on your ship"
+		"imprison anyone they catch with their outfits"
+		"will put you behind bars forever if you've got one of their ships or outfits without their permission"
+		"won't ever let you step foot on a spaceship again if you've got one of their own ships"
+		"don't take any chances when it comes to a misplaced ship or outfit of theirs"
+	word
+		"."
+	replace
+		"It's said that the Quarg make it pretty clear that they" "It's said that the Quarg"
```

Comment:
Why do we not want:
"It's said that the Quarg make it pretty clear that they"
But:
"The Quarg make it pretty clear that they"
"They make it pretty clear that they"
"I'd love to get my hands on some Quarg tech, but they make it pretty clear that they"
Are fine?

Also, the thing with replacing `?!` with `?`; why not just put an `!` at the end of the lines where you want that?

Reply:
`It's said` suggests a rumour, `make it pretty clear that they` suggests some sort of factual basis.

> Also, the thing with replacing ?! with ?; why not just put an ! at the end of the lines where you want that?

I was lazy.